io.openems.edge.evcs.spelsberg.smart/src/io/openems/edge/evcs/spelsberg/smart/EvseState.java

```diff
@@ -0,0 +1,34 @@
+package io.openems.edge.evcs.spelsberg.smart;
+
+import io.openems.common.types.OptionsEnum;
+
+public enum EvseState implements OptionsEnum {
+	
+	UNDEFINED(-1, "Undefined"),
```

Comment:
// 
See other components

File would change if you execute "Format" on the file or complete folder

Reply:
done